### ISSUE TITLE
CUSA08306 - One Piece: Unlimited World Red - Deluxe Edition

### ISSUE THREAD

**Issue by PistolaoBR** [shadps4-emu/shadps4-game-compatibility#1732]
### Game Name

One Piece: Unlimited World Red - Deluxe Edition

### Game code

CUSA08306

### Game version

v01.00

### Used emulator's version (only released versions are acceptable)

v0.5.0

### Current status

Playable

### Operating System

Windows

### Error

[Core] <Error> stubs.cpp:CommonStub:42: Stub: sceMouseRead (nid: x8qnXqh-tiM) called, returning zero to 0x900378577

### Description

Playable, at 60fps, super fluid

### Screenshots

![image](https://github.com/user-attachments/assets/02bc0962-16ef-429e-84e4-b10e3e5ea10c)
![image](https://github.com/user-attachments/assets/16174f20-6123-475d-9356-fc66c1cd37e5)
![image](https://github.com/user-attachments/assets/469435de-fa4c-409d-b2c8-b08d2463bb9d)

### Log File

[shad_log.txt](https://github.com/user-attachments/files/18281660/shad_log.txt)

**Comment by C4ndyF1sh** [comment]
@PistolaoBR i also thought it was playable at first but it always crashes at the 2nd world

**Comment by PistolaoBR** [comment]
> @PistolaoBR i also thought it was playable at first but it always crashes at the 2nd world

thanks Candy, if that is the case, can you play to the 2nd world and see if the 0.4.0 error persists on the 0.5.0 and post it here... Thanks a lot mate, you must still have the save game right?

**Comment by C4ndyF1sh** [comment]
@PistolaoBR @Xphalnos on 0.5.0 it goes a bit further but still crashes 2 cutscenes later, even when skipping them it crashes

**Comment by Xphalnos** [comment]
> @PistolaoBR @Xphalnos on 0.5.0 it goes a bit further but still crashes 2 cutscenes later, even when skipping them it crashes

Do you mean I have to put it in "ingame"?

**Comment by C4ndyF1sh** [comment]
@PistolaoBR No, I refer to shadps4 0.5.0 and 0.5.1 main, and if I would you guys give a save it won't work since I have a different CUSA version

**Comment by C4ndyF1sh** [comment]
> > @PistolaoBR @Xphalnos on 0.5.0 it goes a bit further but still crashes 2 cutscenes later, even when skipping them it crashes
> 
> 
> 
> Do you mean I have to put it in "ingame"?

Yes
